src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -353,6 +358,11 @@ private void LogScriptIfGiven(string type, string scriptPath)
                     _writer.WriteLine("Invalid app.yaml format");
                 }
             }
+            else
+            {
+                _logger.LogDebug("No app.yaml found");
```

Comment:
@tulikac may we keep only `_writer.WriteLine` or `_writer.LogDebug`?  Same with line 323-352 please

Reply:
Done